### ISSUE TITLE
Unable to run Windows 8 tests inside VS

### ISSUE THREAD

**Issue by ManfredVonKanzel** [xunit/xunit#317]
Observed using xunit.runner.visualstudio NuGet package v2.0.0-rc2-build1043.

Created a unit test library targeting Windows 8.1.
When attempting to run tests, runner complains about missing "xunit.runner.visualstudio.win8" assembly.

Looks very similar to https://github.com/xunit/xunit/issues/277, filed for Windows Phone 8.1.

**Comment by xdaDaveShaw** [comment]
Looks similar to Issue 37 on the xunit.runners.visualstudio repo: https://github.com/xunit/visualstudio.xunit/issues/37

**Comment by bradwilson** [comment]
Can you repro with our integration test project?

https://github.com/xunit/visualstudio.xunit.integration/

Also, did you make a Windows Store Unit Test project or a Class Library? The Visual Studio runner only officially works with the former, not the latter, because it adds the required infrastructure that is mandated by the Visual Studio runner. These instructions are a little out of date because they talk about the old Visual Studio VSIX, but you can replace that with the NuGet package reference to `xunit.runner.visualstudio` after the project is created.

http://blog.novotny.org/2014/08/11/now-shipping-xunit-support-for-windows-8-and-windows-phone-app-8-1/

**Comment by ManfredVonKanzel** [comment]
My test project is a Windows Store Unit Test project, from which I removed the MSTestFramework reference.

I could not reproduce using the integration test project.

I compared again both project files, and found the only difference is my project references 2 additional PCLs: a generic one, and another targeting "universal" apps.
I then added a simple reference to a dumb PCL to the "v2_Win81" project inside the integration solution, and got the same issue:

An exception occurred while invoking executor 'executor://xunit/VsTestRunner2': Could not load file or assembly 'xunit.runner.visualstudio.win8, Version=2.0.0.1043, Culture=neutral, PublicKeyToken=null' or one of its dependencies. The system cannot find the file specified.

Hope this helps.
